packages/services/api/src/modules/target/resolvers/Mutation/updateTargetName.ts

```diff
@@ -48,6 +50,27 @@ export const updateTargetName: NonNullable<MutationResolvers['updateTargetName']
     target: targetId,
   });
 
+  // Audit Log Event
+  const currentUser = await injector.get(AuthManager).getCurrentUser();
+  const allUpdatedFields = JSON.stringify({
+    name: input.name,
+  });
```

Comment:
To `updatedFields` we can pass a `JSON` object and then it can just do serialize the inputs before inserting in the DB.